sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexer.java

```diff
@@ -44,5 +46,6 @@ public void buildOnce(SensorContext context) {
     globalSymbolsStep.execute(files, context);
     long stopTime = System.currentTimeMillis() - startTime;
     LOG.debug("Time to build the project level symbol table: " + stopTime + "ms");
```

Comment:
I was a bit hesitant because I found I'm still using this for debugging, to avoid the overhead of the performance monitoring, but indeed it doesn't make much sense to keep this on master anyway.